website/pages/en/mips-faqs.mdx

```diff
@@ -10,7 +10,7 @@ To support the sunsetting of the hosted service and the migration of all of it's
 
 The MIPs program is an incentivization program for Indexers to support them with resources to index chains beyond Ethereum mainnet and help The Graph protocol expand the decentralized network into a multi-chain infrastructure layer.
 
-The MIPs program has allocated 0.75% of the GRT supply (75M GRT), with 0.5% to reward Indexers who contribute to bootstrapping the network and 0.25% allocated to migration grants for subgraph developers using multi-chain subgraphs.
+The MIPs program has allocated 0.75% of the GRT supply (75M GRT), with 0.5% to reward Indexers who contribute to bootstrapping the network and 0.25% allocated to network grants for subgraph developers using multi-chain subgraphs.
```

Comment:
It seems like "Network Grants" is capitalized elsewhere, should it be here as well?

Reply:
Yes!